Objective-C/CBLDatabase.mm

```diff
@@ -357,7 +357,7 @@ - (BOOL) inBatch: (NSError**)outError usingBlock: (void (NS_NOESCAPE ^)())block
     return [self inBatch: outError usingBlockWithError: ^(NSError **) { block(); }];
 }
 
-- (BOOL) withTransaction: (NSError**)outError usingBlockWithError: (BOOL (NS_NOESCAPE ^)(NSError**))block NS_REFINED_FOR_SWIFT {
+- (BOOL) maybeBatch: (NSError**)outError usingBlockWithError: (BOOL (NS_NOESCAPE ^)(NSError**))block {
```

Comment:
Not sure why `maybe`. The original `withTransaction` is fine too as it's an internal function or could be just `inBatchWithError:`?

Reply:
I liked maybe because it won't batch (transaction is aborted) if the closure returns false at any point